coalib/processes/Processing.py

```diff
@@ -441,13 +472,30 @@ def yield_ignore_ranges(file_dict):
                                            len(file[-1])))
 
 
+def get_file_list(results):
+    """
+    Get the list of files that are affected in the given results.
+
+    :param results: A list of results from which the list of files is to be
+                    extracted.
+    :return:        A list of file paths containing the mentioned list of
+                    files.
+    """
+    files = []
+    for result in results:
+        for code in result.affected_code:
+            files.append(code.file)
```

Comment:
make this a generator expression and a set, way faster! `return {code.file for ... for ...}`